Error either with https://securityigert.uic.edu/ or https://www.uis.edu/
Error: ('Connection aborted.', FileNotFoundError(2, 'No such file or directory'))

Error either with https://www.cmhsrp.uic.edu/nrtc/ or https://dscc.uic.edu/terms-of-use/
Error: ('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None))

Error either with https://hpsc.org.uic.edu/ or https://www.uis.edu/
Error: ('Connection aborted.', FileNotFoundError(2, 'No such file or directory'))

Error either with https://nlp.cs.uic.edu/ or https://www.uis.edu/
Error: HTTPSConnectionPool(host='nlp.cs.uic.edu', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x00000231363C63D0>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))

Error either with https://projectheal.uic.edu/ or https://www.uis.edu/
Error: HTTPSConnectionPool(host='projectheal.uic.edu', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002312F3A6E50>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))

Error either with https://tigger.uic.edu/htbin/codewrap/bin/orgs/ura/cgi-bin/ or https://www.uis.edu/
Error: HTTPSConnectionPool(host='tigger.uic.edu', port=443): Max retries exceeded with url: /htbin/codewrap/bin/orgs/ura/cgi-bin/ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000023134FA40D0>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))

Error either with https://www.ecc.uic.edu/ecc/webhome/ or https://www.vpaa.uillinois.edu/resources/web_privacy/
Error: HTTPSConnectionPool(host='ecc.engr.uic.edu', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000023136DFA610>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))

Error either with https://www.uic.edu/depts/oaa/wisest/ or https://www.uis.edu/
Error: HTTPConnectionPool(host='wisest.uic.edu', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000023136D7FD90>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))

Error either with https://cts.cs.uic.edu/ or https://www.acm.org/sigkdd/
Error: HTTPSConnectionPool(host='cts.cs.uic.edu', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x0000023138705190>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))

Error either with https://gila.bioe.uic.edu/lab/ or https://www.uis.edu/
Error: HTTPSConnectionPool(host='gila.bioe.uic.edu', port=443): Max retries exceeded with url: /lab/ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002313999A220>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))

Error either with https://sts.bioe.uic.edu/castp/ or https://www.uis.edu/
Error: HTTPSConnectionPool(host='sts.bioe.uic.edu', port=443): Max retries exceeded with url: /castp/ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002313999AA00>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))

Error either with https://healthcheck.uic.edu/ or https://uis.edu/
Error: HTTPSConnectionPool(host='healthcheck.uic.edu', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002313996D2B0>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))

Error either with https://go.uic.edu/dailypass/ or 
Error: HTTPSConnectionPool(host='healthcheck.uic.edu', port=443): Max retries exceeded with url: /AAMEmployeeOnsiteStatus/Home/CheckStatus/ (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002313A697670>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))

Error either with https://go.uic.edu/dailysurvey/ or https://www.uis.edu/
Error: HTTPSConnectionPool(host='healthcheck.uic.edu', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x000002313C76BCD0>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))

